OpenDreamShared/Dream/AtomDirection.cs

```diff
@@ -2,7 +2,7 @@
 using System;
 
 namespace OpenDreamShared.Dream {
-    [Serializable, NetSerializable]
+    [Serializable, NetSerializable, Flags]
```

Comment:
Not sure why this wasn't applied.

Reply:
Wasn't aware of its existence when I first wrote this